bitnami/keycloak/templates/admin-ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.adminIngress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
```

Comment:
It's only when I tried to render the template with the `helm` command that I realised that there was an error in my code: I wrote `include "common.capabilities.adminIngress.apiVersion"` instead of `include "common.capabilities.ingress.apiVersion"`.
I just blindly copy/pasted from the original Ingress and then did find/replace 😅 

But that made me think: couldn't an automated test have caught this kind of error earlier on?
It's out of scope of this MR, but isn't it the case that no such tests are being run for Bitnami charts in CI?
Like for example trying to simply render the chart, at least with the default values, but also maybe why not against a set of manually crafted values files?

Reply:
Doh! I should have done my homework regarding my last remark.
There *is* indeed this "VIB Verify" step in the CI.
Which, from what I see inside the artifact, does make an actual deployment on Kubernetes:
```
0390aaed-a3be-4824-bd32-a918edfcd1df-deployment-logs.txt
0390aaed-a3be-4824-bd32-a918edfcd1df-deployment-report.json
11d86635-6bd5-4fd5-a79a-9cefd616b152-helm-lint-logs.txt
11d86635-6bd5-4fd5-a79a-9cefd616b152-helm-lint-raw_report-helm-linter-output.txt
11d86635-6bd5-4fd5-a79a-9cefd616b152-helm-lint-report.json
58b9f92d-1b59-4754-8fc7-dd4037ee2b43-deployment-logs.txt
58b9f92d-1b59-4754-8fc7-dd4037ee2b43-deployment-report.json
788b2936-0c8a-4b27-b5e2-177b0815b50e-provisioning-report.json
8d1182f5-04d5-4dc2-9d56-1e025e25f7cd-cypress-logs.txt
8d1182f5-04d5-4dc2-9d56-1e025e25f7cd-cypress-raw_report-keycloak-spec-js.xml
8d1182f5-04d5-4dc2-9d56-1e025e25f7cd-cypress-report.json
a54de195-60b5-4134-aae8-770b2b3a377e-package-scanner-logs.txt
a54de195-60b5-4134-aae8-770b2b3a377e-package-scanner-report.json
da731e80-7b2c-413c-bec2-7e9939e4cba9-helm-package-logs.txt
da731e80-7b2c-413c-bec2-7e9939e4cba9-helm-package-report.json
f0df7cdb-3e09-4a5c-ba03-707011d9b829-goss-logs.txt
f0df7cdb-3e09-4a5c-ba03-707011d9b829-goss-raw_report-keycloak-goss-goss-yaml-amd64.xml
f0df7cdb-3e09-4a5c-ba03-707011d9b829-goss-report.json
README.md
report.json
```

I suppose the deployment uses the default `values.yaml`, where this admin Ingress is not enabled by default, which would explain why the issue I referred to in my message above wasn't discovered during CI.

Reply:
Thanks a lot for the pointer 👍